README.md

```diff
@@ -94,6 +95,18 @@ To see this report while running tests locally, type:
 
     open ./coverage/index.html
 
+### Running development AMO similar to production
```

Comment:
These docs look good, thanks, but when the commands get merged this will need some re-organizing. The no-proxy section should explicitly mention how to connect to `addons-dev.allizom.org`.